src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
```

Comment:
There are a few directions we can go:
- too much defensive coding is fluff, don't call main with null (potential bug maybe)
- mark with @NonNull
- use guava precondition assertion to check for null
- approach taken,  check for null and set it to non-null

Probably better to handle the situation in one way or another, "fail fast" principle. I could go almost any way on it, probably given it has raised conversation, better to handle the potential error condition in one or another (thinking about the fact that we call main from multiple places, not just command line)